BUILD.bazel

```diff
@@ -59,3 +59,23 @@ xcode_config(
     ],
     visibility = ["//visibility:public"],
 )
+
+xcode_version(
+  name = 'version12_5_1_12E507',
+  version = '12.5.1.12E507',
+  aliases = ['12.5.1' ,'12' ,'12.5' ,'12.5.1.12E507'],
+  default_ios_sdk_version = '14.5',
+  default_tvos_sdk_version = '14.5',
+  default_macos_sdk_version = '11.3',
+  default_watchos_sdk_version = '7.4',
+)
+
+xcode_config(name = 'host_xcodes',
+  versions = [':version12_5_1_12E507'],
```

Comment:
I think this change is causing a problem with the existing `host_xcodes` -  `version12_5_1_12E507` should be added to `versions` in the existing `host_xcodes`